tests/optim/utils/test_circuits.py

```diff
@@ -10,41 +10,70 @@
 
 class TestGetExpandedWeights(BaseTest):
     def test_get_expanded_weights(self) -> None:
-        if torch.__version__ == "1.2.0":
+        if torch.__version__ <= "1.2.0":
             raise unittest.SkipTest(
                 "Skipping get_expanded_weights test due to insufficient Torch version."
             )
-        model = googlenet(pretrained=True)
+        model = googlenet(pretrained=True, use_linear_modules_only=True)
         output_tensor = circuits.get_expanded_weights(
             model, model.mixed3a, model.mixed3b
         )
         self.assertTrue(torch.is_tensor(output_tensor))
         self.assertEqual(list(output_tensor.shape), [480, 256, 28, 28])
 
     def test_get_expanded_weights_crop_int(self) -> None:
-        if torch.__version__ == "1.2.0":
+        if torch.__version__ <= "1.2.0":
             raise unittest.SkipTest(
                 "Skipping get_expanded_weights crop test due to insufficient Torch"
                 + " version."
             )
-        model = googlenet(pretrained=True)
+        model = googlenet(pretrained=True, use_linear_modules_only=True)
         output_tensor = circuits.get_expanded_weights(
             model, model.mixed3a, model.mixed3b, 5
         )
         self.assertEqual(list(output_tensor.shape), [480, 256, 5, 5])
 
     def test_get_expanded_weights_crop_two_int(self) -> None:
-        if torch.__version__ == "1.2.0":
+        if torch.__version__ <= "1.2.0":
             raise unittest.SkipTest(
                 "Skipping get_expanded_weights two int crop test due to insufficient"
                 + " Torch version."
             )
-        model = googlenet(pretrained=True)
+        model = googlenet(pretrained=True, use_linear_modules_only=True)
         output_tensor = circuits.get_expanded_weights(
             model, model.mixed3a, model.mixed3b, (5, 5)
         )
         self.assertEqual(list(output_tensor.shape), [480, 256, 5, 5])
 
+    def test_get_expanded_nonlinear_top_connections(self) -> None:
+        if torch.__version__ <= "1.2.0":
+            raise unittest.SkipTest(
+                "Skipping get_expanded_weights nonlinear_top_connections test"
+                + " due to insufficient Torch version."
+            )
+
+        if torch.__version__ == "1.3.0":

```

Comment:
is this `torch.__version__ == "1.3.0"` or `torch.__version__ <= "1.3.0"`  ?

Reply:
Oops, yeah I forgot to add the less than sign!